lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -432,47 +432,41 @@ def operations
       end
 
       define_method("find_all_of_type_#{pe_type}") do |options = {}|
+        options[:unique_identifier] = options.delete(:uuid) if options[:uuid]
         conditions = options.slice!(:per_page, :page, :ignore_case).stringify_keys
+        # Separate conditions on PolicyElement columns from conditions on the extra_attributes column
         extra_attribute_conditions = conditions.slice!(*PolicyElement.column_names)
         include_conditions, conditions = conditions.partition { |k,v| include_condition?(k,v) }
+        # Generated PolicyElement class
```

Comment:
Nitpick: we can increase readability a bit by adding some returns here and above.